packages/utils/find-root.ts

```diff
@@ -0,0 +1,8 @@
+import * as findup from "findup-sync";
```

Comment:
We are almost there! So, the `findup-sync` package is in the root project but is not installed in the utils project.

So you have two alternatives:
- (the quick one) is to add the `findup-sync` package also here in the utils project
- (the better one) remove the `findup-sync` package from the root project, add it in the utils project and **expose a utility function to fix the usage of package in the root project**.

Choose the solution that you prefer. I'd prefer the second but it can tackled in a second PR

Reply:
Could we go for no.2?

Reply:
Yes, added the package in utils for now, doing it in a new PR asap.

@evenstensberg @ematipico

Reply:
Hi @ematipico 

I created util in `packages/utils` but then what's the workflow to import it in the root project?

Reply:
One way could be to import it from `@webpack-cli/utils/fileName` but for it we'll need to publish it right?